tools/gvm-migrate-config

```diff
@@ -2537,8 +2537,8 @@ cat > ${TMP}/migrate.xsl << 'EOF'
     <preference><oid>1.3.6.1.4.1.25623.1.0.104118</oid><id>2</id><name>smbport</name></preference>
     <preference><oid>1.3.6.1.4.1.25623.1.0.104118</oid><id>1</id><name>smbbasic</name></preference>
     <preference><oid>1.3.6.1.4.1.25623.1.0.96180</oid><id>4</id><name>Target checksum File</name></preference>
-    <preference><oid>1.3.6.1.4.1.25623.1.0.96180</oid><id>3</id><name>Delete hash test Programm after the test</name></preference>
-    <preference><oid>1.3.6.1.4.1.25623.1.0.96180</oid><id>2</id><name>Install hash test Programm on the Target</name></preference>
+    <preference><oid>1.3.6.1.4.1.25623.1.0.96180</oid><id>3</id><name>Delete hash test Program after the test</name></preference>
+    <preference><oid>1.3.6.1.4.1.25623.1.0.96180</oid><id>2</id><name>Install hash test Program on the Target</name></preference>
```

Comment:
AFAIK these are typos in that specific VT which currently can't be changed as long as a migration to GOS 6.0 is still supported.